rudder-agent/SOURCES/patches/cfengine/02-sighup-support-in-cfserverd.patch

```diff
@@ -1,82 +0,0 @@
---- a/cf-serverd/cf-serverd-functions.c	2014-11-26 06:26:51.000000000 +0100
-+++ b/cf-serverd/cf-serverd-functions.c	2014-11-28 15:10:21.624969933 +0100
-@@ -356,6 +356,14 @@
-     Log(LOG_LEVEL_DEBUG, "Checking file updates for input file '%s'",
-         config->input_file);
- 
-+    /* If we force the reload of config, effectively recheck the promises */
-+    if ( IsRequestReloadConfig() )
-+    {
-+        Log(LOG_LEVEL_VERBOSE, "Force reload of inputs files...");
-+        GenericAgentCheckPolicy(config, true, true);
```

Comment:
The original PR is https://github.com/cfengine/core/pull/2008, the code finally introducing the change is https://github.com/cfengine/core/commit/d0547c25247a7a9fec07585e2e2f93a27b98b8cb. It uses the same code as auto-reload, so I guess It should work as expected.